store/postgres/src/relational.rs

```diff
@@ -1067,6 +1068,11 @@ impl Column {
         named_type(&self.field_type) == "String" && !self.is_list()
     }
 
+    // Checks if this column's type is a binary string (Posgres' `bytea`)
+    pub fn is_binary_string(&self) -> bool {
```

Comment:
I'm not fully confident that this is the right way of checking this but it seems to work

@lutter @leoyvens